lib/services/ios-project-service.ts

```diff
@@ -138,19 +145,12 @@ class IOSProjectService implements  IPlatformProjectService {
 					'VALID_ARCHS=armv7 arm64',
 					"CONFIGURATION_BUILD_DIR=" + path.join(projectRoot, "build", "device")
 				]);
-			} else {
-				args = basicArgs.concat([
-					"-sdk", "iphonesimulator",
-					"-arch", "i386",
-					"VALID_ARCHS=\"i386\"",
-					"CONFIGURATION_BUILD_DIR=" + path.join(projectRoot, "build", "emulator")
-				]);
 			}
 
 			this.$childProcess.spawnFromEvent("xcodebuild", args, "exit", {cwd: options, stdio: 'inherit'}).wait();
 
-			if(options.device) {
-				var buildOutputPath = path.join(projectRoot, "build", options.device ? "device" : "emulator");
+			if(!options.emulator) {
+				var buildOutputPath = path.join(projectRoot, "build", options.emulator ? "emulator" : "device");
```

Comment:
we get here only when options.emulator is false (undefined), why do we check again, shouldn't we use "device"directly?